test/controllers/organization_controller_test.exs

```diff
@@ -0,0 +1,102 @@
+defmodule CodeCorps.OrganizationControllerTest do
+  use CodeCorps.ConnCase
+
+  alias CodeCorps.Organization
+  alias CodeCorps.Repo
+
+  @valid_attrs %{description: "Build a better future.", name: "Code Corps", slug: "code-corps"}
+  @invalid_attrs %{}
+
+  setup do
+    conn = %{build_conn | host: "api."}
+      |> put_req_header("accept", "application/vnd.api+json")
```

Comment:
Thought this looked a little weird and saw that on [this Elixir style guide](https://github.com/niftyn8/elixir_style_guide) and [this one](https://github.com/rrrene/elixir-style-guide) the syntax should look like:
```
sanitized_string =
  some_string
  |> String.downcase
  |> String.strip
```